lib/api/_loaders/ensure.js

```diff
@@ -260,8 +260,10 @@ class EnsureAssertionLoader extends BaseLoader {
         namespace = parent.ensure = parent.ensure || {};
       }
 
+      const abortOnFailure = this.nightwatchInstance.settings.globals.abortOnAssertionFailure;
+
       this.nightwatchInstance.setApiMethod(propertyName, namespace || 'ensure', (function(commandName) {
-        return this.createAssertion(commandName, false);
+        return this.createAssertion(commandName, abortOnFailure);
```

Comment:
I added this assuming ensure should behave like normal asserts.

Reply:
We would need to change documentation as well if we plan to consider this.